Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs

```diff
@@ -70,7 +71,8 @@ private void OnMapInit(EntityUid uid, ItemSlotsComponent itemSlots, MapInitEvent
                 if (slot.HasItem || string.IsNullOrEmpty(slot.StartingItem))
                     continue;
 
-                var item = EntityManager.SpawnEntity(slot.StartingItem, EntityManager.GetComponent<TransformComponent>(uid).Coordinates);
+               var item = Spawn(slot.StartingItem, Transform(uid).Coordinates);
```

Comment:
I think it got misaligned by one space there.
```suggestion
                var item = Spawn(slot.StartingItem, Transform(uid).Coordinates);
```